includes/REST_API/Stories_Users_Controller.php

```diff
@@ -94,6 +94,28 @@ public static function get_registration_action_priority(): int {
 		return 100;
 	}
 
+	/**
+	 * Permissions check for getting all users.
+	 *
+	 * Allows edit_posts capabilities queries for stories if the user has the same cap,
+	 * enabling them to see the users dropdown.
+	 *
+	 * @since 1.28.1
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return true|WP_Error True if the request has read access, otherwise WP_Error object.
+	 */
+	public function get_items_permissions_check( $request ) {
+		if (
+			! empty( $request['capabilities'] ) &&
+			[ $this->story_post_type->get_cap_name( 'edit_posts' ) ] === $request['capabilities']
+		) {
+			unset( $request['capabilities'] );
```

Comment:
`capabilities` can be an array right? Should we not just unset the element in array. For example if you request this. 

/web-stories/v1/users?capabilities[]=test&capabilities[]=edit_posts

What about that case?

Reply:
> `capabilities` can be an array right?

Right, which is why `[ $this->story_post_type->get_cap_name( 'edit_posts' ) ] === $request['capabilities']` compares an array with an array (note the brackets)

If any other caps are in the array, this will continue to return a permission error